tests/pybind11_cross_module_tests.cpp

```diff
@@ -104,6 +104,7 @@ PYBIND11_MODULE(pybind11_cross_module_tests, m) {
     m.def("return_self", [](LocalVec *v) { return v; });
     m.def("return_copy", [](const LocalVec &v) { return LocalVec(v); });
 
+    // NOLINTNEXTLINE
     class Dog : public pets::Pet { public: Dog(std::string name) : Pet(name) {}; };
```

Comment:
@rwgk One of the problematic lines of code. FIxing this with `: Pet(std::move(name))` would break things.